storage/repair.go

```diff
@@ -71,18 +68,16 @@ func newShardRepairer(opts Options, rpopts repair.Options) (databaseShardRepaire
 	}
 
 	iopts := opts.InstrumentOptions()
-	scope := iopts.MetricsScope().SubScope("database.repair").Tagged(map[string]string{"host": hostname})
-	rtopts := opts.RetentionOptions()
+	scope := iopts.MetricsScope().SubScope("repair").
+		Tagged(map[string]string{"host": hostname})
```

Comment:
Hm we should probably get rid of this.  We specify whether to include the host all the way at the very beginning when we create a scope, seems strange to override/add it there. 

I think we should remove the hostname tagging altogether.